lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -29,11 +30,21 @@ module RSpec
       #   # good
       #   it 'does things' do
       #   end
+      #
+      # @example
```

Comment:
It would be good to have a good case/bad case according to the setting values of the configuration options.

Reply:
I think I addressed it with my latest changes, but let me know if you were expecting something else!

Reply:
I think the part you added is an explanation of the case where 'it' is repeated. I thinking of the https://github.com/rubocop/rubocop-rspec/pull/1350#discussion_r938271855 changes. For your information.